src/github.js

```diff
@@ -315,6 +315,12 @@
          this.createRepo = function (options, cb) {
             _request('POST', '/user/repos', options, cb);
          };
+
+         // Create an Organization repo
+         // -------
+         this.createOrgRepo = function (orgname, options, cb) {
```

Comment:
Instead of adding a new parameter, I think we can pass the organization name as part of the options.